scylla_test.go

```diff
@@ -167,6 +169,9 @@ func TestScyllaRandomConnPIcker(t *testing.T) {
 	})
 
 	t.Run("async access of max iterations", func(t *testing.T) {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
On a busy system timeout as low as this could flake. Is there a particular issue you've encountered that forced you to start addressing this? Would collecting the errors be enough to fix the vet?